blocks/carousel/carousel.css

```diff
@@ -140,19 +141,42 @@ main .hero-slider .carousel-item-text {
 }
 
 /* hero slider text */
-main .carousel-item-text p {
+ .hero-slider .carousel-item-text p {
     margin: 10px 0 16px;
     line-height: 1.2;
     font-weight: 400;
     color: white;
 }
 
 /* hero slider text */
-main .carousel-item-text h2 {
+ .hero-slider .carousel-item-text h2 {
     margin: 0;
     margin-top: 16px;
     font-weight: 200;
     word-break: break-word;
     overflow-wrap: break-word;
     color: white;
 }
+
+@media (max-width: 768px) {
```

Comment:
please don't use the 768 breakpoint, but use the 600,900,1200 instead.
there is an interesting thread about it here: https://cq-dev.slack.com/archives/C9KD0TT6G/p1679320211506049